config/tutorials/hdfs-logs/index-config.yaml

```diff
@@ -20,6 +20,7 @@ doc_mapping:
     - name: severity_text
       type: text
       tokenizer: raw
+      fast: true
     - name: body
       type: text
       tokenizer: default
```

Comment:
For the lambda, we should limit the `split_num_docs_target` to 2 million otherwise, we won't benefit from parallelization.
I indexed the dataset and got a split of 14 million documents. The aggregation takes 1400ms on this split...